modules/op-release-notes-1-8.adoc

```diff
@@ -533,3 +533,69 @@ securityContext:
 // link:https://issues.redhat.com/browse/SRVKP-1806
 // Rupali Behera
+
+
+[id="release-notes-1-8-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.8.1
+
+With this update, {pipelines-title} General Availability (GA) 1.8.1 is available on {product-title} 4.10 and 4.11.
+
+[id="known-issues-1-8-1_{context}"]
+=== Known issues
+
+* When installer sets are in failed state, the status of the `TektonConfig` custom resource is incorrectly displayed as `True` instead of `False`.
++
+.Example: Failed installer sets
+[source,terminal]
+----
+$ oc get tektoninstallerset 
+NAME                                     READY   REASON
+addon-clustertasks-nx5xz                 False   Error
+addon-communityclustertasks-cfb2p        True    
+addon-consolecli-ftrb8                   True    
+addon-openshift-67dj2                    True    
+addon-pac-cf7pz                          True    
+addon-pipelines-fvllm                    True    
+addon-triggers-b2wtt                     True    
+addon-versioned-clustertasks-1-8-hqhnw   False   Error
+pipeline-w75ww                           True    
+postpipeline-lrs22                       True    
+prepipeline-ldlhw                        True    
+rhosp-rbac-4dmgb                         True    
+trigger-hfg64                            True    
+validating-mutating-webhoook-28rf7       True 
+----
++
+.Example: Incorrect `TektonConfig` status
+[source,terminal]
+----
+$ oc get tektonconfig config
+NAME     VERSION   READY   REASON
+config   1.8.1     True 
+----
+
+// https://issues.redhat.com/browse/SRVKP-2556
+
+[id="fixed-issues-1-8-1_{context}"]
+=== Fixed issues
+
+* Before this update, the pruner deleted task runs of running pipelines and displayed the following warning: `some tasks were indicated completed without ancestors being done`. With this update, the pruner retains the task runs that are part of running pipelines.
+// https://issues.redhat.com/browse/SRVKP-2419
+
+* Before this update, `pipeline-1.8` was the default channel for installing the {pipelines-title} Operator 1.8.x. With this update, `latest` is the default channel.
+// https://issues.redhat.com/browse/SRVKP-2412
+
+* Before this update, the {pac} controller pods did not have access to certificates exposed by the user. With this update, {pac} can now access routes and Git repositories guarded by a self-signed or a custom certificate.
+// https://issues.redhat.com/browse/SRVKP-2470
+
+* Before this update, the task failed with RBAC errors after upgrading from {pipelines-title} 1.7.2 to 1.8.0. With this update, the tasks run successfully without any RBAC errors.
+// https://issues.redhat.com/browse/SRVKP-2472
+
+* Before this update, using the `tkn` CLI tool you could not remove the task runs and pipline runs containing a `result` object of `array` type. With this update, you can use the `tkn` CLI tool to remove the task runs and pipeline runs containing a `result` object of `array` type.
```

Comment:
As someone who probably over-uses commas, I don't have a problem leaving out the comma in "using the `tkn` CLI tool you could not" as I think it helps the flow when reading.

I don't have a problem with "the task runs... containing" - using the past tense seems more stilted to me

I was tempted to suggest "object of type `array`" as that would be pretty common usage in developer-specific content, but I thought it might be nit-picking.

I know there is a preference for past tense in RNs, but "that contained....whose type was..."   followed by "that contain....whose type is"  just feels a lot more difficult to parse that having parallel "containing....of type" in both places. 

Yes, I did miss pipline - I will copy all content into an editor in future to spell-check - lesson learned.

Reply:
While I was commenting on a spelling issue ("piplines") and a missing comma at the end of a parenthetical phrase ("tool you"), I included some style preferences that I believe improve clarity (all the other suggestions). You're welcome to take or leave what you want.